test/navigation/nonsipp/OtherAssetsDisposalNavigatorSpec.scala

```diff
@@ -127,10 +127,27 @@ class OtherAssetsDisposalNavigatorSpec extends BaseSpec with NavigatorBehaviours
             assetIndex,
             disposalIndex,
             WhenWasAssetSoldPage,
+            (srn, assetIndex: Max5000, disposalIndex: Max50, _) =>
+              controllers.nonsipp.otherassetsdisposal.routes.TotalConsiderationSaleAssetController
+                .onPageLoad(srn, assetIndex, disposalIndex, NormalMode)
+          )
+          .withName("go from when was asset sold page to WhenWasAssetSold page")
```

Comment:
Is this name right? If not, it can be changed with the next change

Reply:
Ok Kornel, will change in my next ticket